include/SFML/Window/Event.hpp

```diff
@@ -100,7 +100,7 @@ class Event
     };
 
     ////////////////////////////////////////////////////////////
-    /// \brief Mouse wheel events parameters (MouseWheelMoved)
+    /// \brief Mouse wheel events parameters (MouseWheelMoved) (deprecated)
```

Comment:
I guess the typical usage would be (on a new line): `\deprecated this struct is deprecated`. (See bottom of [this page](https://code.google.com/p/screen3d/wiki/Doxygen).)